### PR TITLE
BIT-2398 if the org associated with a cipher uses TOTP enable the aut…

### PR DESCRIPTION
**Pull request by dseverns-livefront**
## 🎟️ Tracking

[BIT-2398](https://livefront.atlassian.net/jira/software/c/projects/BIT/boards/91?assignee=712020%3A179d6d43-0428-44b1-82e0-e454ccab0c92&selectedIssue=BIT-2398)

## 📔 Objective
When there are features around TOTP, in addition to having premium, accounts which have a cipher which had TOTP enabled by an Org. should also be considered

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Comment by dseverns-livefront** [comment]
**Note** [This](https://github.com/bitwarden/ios/pull/696/files) is the iOS PR for similar ticket, the changes there looks a lot close to the legacy app. I took this to be the way we are storing and retrieving the ciphers from disk. This was the only area where I found `isPremium` considered along with a `TOTP` feature.

File: app/src/test/java/com/x8bit/bitwarden/data/autofill/manager/AutofillCompletionManagerTest.kt [hunk 1]
```diff
@@ -311,6 +311,98 @@ class AutofillCompletionManagerTest {
...
+        autofillCompletionManager.completeAutofill(
+            activity = activity,
+            cipherView = cipherView,
+        )
+
+        verify {
+            activity.setResult(Activity.RESULT_OK, resultIntent)
+            activity.finish()
+        }
+        verify {
```
Comment (david-livefront) [inline app/src/test/java/com/x8bit/bitwarden/data/autofill/manager/AutofillCompletionManagerTest.kt:371]: Why split up the verify blocks?

File: app/src/test/java/com/x8bit/bitwarden/data/autofill/manager/AutofillCompletionManagerTest.kt [hunk 2]
```diff
@@ -459,6 +551,7 @@ class AutofillCompletionManagerTest {
             )
         } returns fillableRequest
         every { cipherView.login?.totp } returns TOTP_CODE
+        every { cipherView.organizationUseTotp } returns false
```
Comment (david-livefront) [inline app/src/test/java/com/x8bit/bitwarden/data/autofill/manager/AutofillCompletionManagerTest.kt:554]: Wanna add the verify for this too

File: app/src/test/java/com/x8bit/bitwarden/data/autofill/manager/AutofillCompletionManagerTest.kt (hunk 1 above)
Comment (dseverns-livefront) [reply app/src/test/java/com/x8bit/bitwarden/data/autofill/manager/AutofillCompletionManagerTest.kt:371]: cmd c + cmd v from the test above, tldr laziness

Comment (david-livefront) [reply app/src/test/java/com/x8bit/bitwarden/data/autofill/manager/AutofillCompletionManagerTest.kt:371]: Ohh, weird

**Review by david-livefront — DISMISSED** [review]

**Review by SaintPatrck — APPROVED** [review]

**Pull request merged**